internal/commands/serverfirewall/create.go

```diff
@@ -51,6 +52,11 @@ func (s *createCommand) MaximumExecutions() int {
 
 // InitCommand implements Command.InitCommand
 func (s *createCommand) InitCommand() {
+	s.Cobra().Long = `Create a new firewall rule
+
+To edit the default rule of the firewall, set only --direction and --action parameters.
+This will create catch-all rule that will take effect when no other rule matches.`
```

Comment:
Is this accurate/clear-enough?

Reply:
Almost. The default rule should also be the last rule in the whole set. So it has to be either created last, or it has to be placed at the end using the `--position` flag